.github/Makefile

```
@@ -39,6 +39,7 @@ configure: n98-magerun2.phar
 	bin/magento config:set payment/adyen_giving/background_image ''
 	bin/magento config:set payment/adyen_abstract/merchant_account "${ADYEN_MERCHANT}"
 	bin/magento config:set payment/adyen_moto/active 1
+	bin/magento config:set payment/adyen_abstract/notifications_ip_check 1
```

We can disable this check in CI: 

```suggestion
	bin/magento config:set payment/adyen_abstract/notifications_ip_check 0
```